src/components/SelectionCart/SelectionCart.js

```diff
@@ -1,23 +1,129 @@
+            case types.ADD_QUANTITY:
+                this.props.addQuantity(component);
+                break;
+            case types.SUB_QUANTITY:
+                this.props.subtractQuantity(component);
+                break;
+            case types.REMOVE_COMPONENT:
+                this.props.removeItem(component);
+                break;
+            default:
+                break;
+        }
+        this.setState({ handleChange: !this.state.handleChange });
+    }
+
+
+    getItems() {
+        if (!this.props.addedItems)
+            return;
+
+        let res = [];
+        let item = null, section_ = null;
+        for (let component in this.props.addedItems) {
+            section_ = this.props.addedItems[component].section;
+            if (!this.props.components.hasOwnProperty(section_)) {
+                continue;
+            }
+            item = this.props.components[section_][component];
+            if (typeof item === 'undefined') {
+                continue;
+            }
+            res.push(
+                <Row key={component} className='justify-content-center'>
+                    <Col xs='3'>
+                        <img className='component-img' alt={item.name} src={this.tryRequire(section_, item.img_path)} />
+                    </Col>
+                    <Col xs='5'>
+                        {item.name}
+                    </Col>
+                    <Col xs='4'>
+                        <Row>
+                            <Col xs='2'>
+                                <button onClick={() => this.handleAction(types.SUB_QUANTITY, component)}>-</button>
+                            </Col>
+                            <Col xs='4'>
+                                {this.props.addedItems[component].quantity}
+                            </Col>
+                            <Col xs='2'>
+                                <button onClick={() => this.handleAction(types.ADD_QUANTITY, component)}>+</button>
+                            </Col>
+                            <Col xs='2'>
+                                <button onClick={() => this.handleAction(types.REMOVE_COMPONENT, component)}>x</button>
+                            </Col>
+                        </Row>
+                    </Col>
+                </Row>
+            );
+        }
+        return res;
     }
 
     render() {
-        return(
-            <div>
-                <ul>Component 1</ul>
-                <ul>Component 2</ul>
-                <ul>Component 3</ul>
-                <ul>Component 4</ul>
-            </div>
+        return (
+            <Col className='cart-container'>
+                <Row className='cart-header'>
+                    Checkout!
+                </Row>
+                <Col className='cart-collection'>
+                    {this.getItems()}
+                </Col>
+                <Row className='justify-content-center'>
+                    <button
+                        onClick={this.props.clearCart}>
```

Comment:
When you click 'Done' for the first time it works fine, but when you click on it after the checkout has been completed the button does nothing. Maybe try to add a function to close the modal once the button 'Done' has been clicked after the checkout or just hide the button completely.